nucliadb/nucliadb/writer/api/v1/upload.py

```diff
@@ -834,7 +834,7 @@ async def store_file_on_nuclia_db(
     md5: Optional[str] = None,
     item: Optional[CreateResourcePayload] = None,
     wait_on_commit: bool = False,
-) -> int:
+) -> Optional[int]:
```

Comment:
How come there's no sequence ids with processing v2?

Reply:
Because it's not an ordered queue like that. I could add a seqid but it would have no bearing on when something was actually scheduled/run/etc